arpack-ng-config.cmake.in

```diff
@@ -19,7 +19,7 @@ set(includedir "@includedir@")
 
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack${SUFFIX64}")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack@LIBSUFFIX@")
```

Comment:
Like a wrote in the downstream PR, this might be an unrelated issue. I opened PR #333 for that.